app/assets/test.html

```diff
@@ -0,0 +1,28 @@
+<html>
```

Comment:
What is this file, and could its use be better documented (perhaps through a more descriptive filename)?

Reply:
This snuck in, should be deleted.

Reply:
And now it is.